src/lib/testPaymentExpiry.js

```diff
@@ -37,12 +37,6 @@ TransferTester.prototype.isAtomic = function () {
   return getTransferCases(this.source_transfers[0]).length > 0
 }
 
-TransferTester.prototype.isFinal = function () {
-  return !_.some(this.destination_transfers, function (transfer) {
-    return transfer.hasOwnProperty('execution_condition')
-  })
-}
-
```

Comment:
https://github.com/interledger/five-bells-connector/blob/feature/st-new-cc/src/lib/testPaymentExpiry.js#L113-L147 is also about using the last ledger's transfer state receipt as the condition